pkgs/tools/filesystems/e2fsprogs/default.nix

```diff
@@ -34,12 +35,11 @@ stdenv.mkDerivation rec {
     ];
 
   checkInputs = [ perl ];
-  doCheck = false; # fails
+  doCheck = true;
 
-  # hacky way to make it install *.pc
   postInstall = ''
-    make install-libs
-    rm "$out"/lib/*.a
+    # avoid cycle between outputs
+    mv $out/lib/${pname}/e2scrub_all_cron $bin/bin/
```

Comment:
Does anything shipped by the package _use_ that program? That looks like a change that should break references…

Reply:
It doesn't appear to be-- we don't `install-crontab` which would be the only reference to it AFAICT.  Perhaps we could remove it but it doesn't hurt anything by keeping it either... :)